api/pkg/store/mongo/store.go

```diff
@@ -13,6 +14,10 @@ import (
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
+var wrongType error = errors.New("wrong parameters type")
+
+const match string = "$match"
```

Comment:
The idea is good but it mixes string and consts. I'd probably avoid adding this const so the query uses a single style.